giraffe/src/utils/legend/useTooltipStyle.ts

```diff
@@ -125,13 +132,23 @@ export const useAnnotationStyle = (
         }
       }
 
-      const clampedX = Math.round(Math.max(dx + x, x))
+      let clampedX = Math.round(x + dx)
       const clampedY = Math.round(dy + y)
 
+      // When the annotation is in the far edge of the screen, the position.left value
+      // overrides the width of the tooltip and makes its width smaller than its max-width.
+      // Fix the left value so that the tooltip is at its max width.
```

Comment:
```suggestion
      // Position the left edge of the tooltip such that the tooltip occupies its max width.
```
pedantic, but reading over this comment this morning (i find that reading stuff the day after i wrote it gives me a much fresher perspective on how it reads to someone else), i think it could be a tad bit clearer